docs/source/workflow/build-run-queries.mdx

```diff
@@ -2,134 +2,150 @@
 
 ## Changing the landing page
 
-You can change the landing page that's served from Apollo Server's base URL. In addition to [configuring the default landing page](#configuring-the-default-landing-page), you can serve [GraphQL Playground](#graphql-playground) (an open-source GraphQL IDE), a completely [custom landing page](#custom-landing-page), or you can [disable the landing page entirely](#disabling-the-landing-page).
+You can change the landing page that's served from Apollo Server's base URL. You can choose to [configure the default landing page](#configuring-the-default-landing-page), serve [GraphQL Playground](#graphql-playground) (an open-source GraphQL IDE), create a completely [custom landing page](#custom-landing-page), or you can [disable the landing page entirely](#disabling-the-landing-page).
 
 You do this by installing one of a few different Apollo Server **plugins**.
 
 ### Configuring the default landing page
 
 Apollo Server uses one of these plugins to display its default landing page, depending on the environment:
 
-* `ApolloServerPluginLandingPageLocalDefault` (non-production environments)
-* `ApolloServerPluginLandingPageProductionDefault` (production)
+- `ApolloServerPluginLandingPageLocalDefault` (non-production environments)
+- `ApolloServerPluginLandingPageProductionDefault` (production)
 
 You can install either of these plugins manually to configure its behavior.
 
-This sample replicates Apollo Server's default behavior, except it removes the footer that each landing page displays:
+For example, below, we set up our local landing page to serve up a splash page which contains a link to the Apollo Sandbox:
 
-```js {1-4,10-12} title="index.js"
-const {
-  ApolloServerPluginLandingPageProductionDefault
+```ts title="index.ts"
+import { ApolloServer } from '@apollo/server';
+// highlight-start
+import {
   ApolloServerPluginLandingPageLocalDefault,
-} = require('apollo-server-core');
+  ApolloServerPluginLandingPageProductionDefault,
+} from '@apollo/server/plugin/landingPage/default';
+// highlight-end
 
 const server = new ApolloServer({
   typeDefs,
   resolvers,
   plugins: [
-    process.env.NODE_ENV === 'production' ?
-      ApolloServerPluginLandingPageProductionDefault({ footer: false }) :
-      ApolloServerPluginLandingPageLocalDefault({ footer: false })
-  ]
+    // highlight-start
+    process.env.NODE_ENV === 'production'
+      ? ApolloServerPluginLandingPageProductionDefault()
+      : ApolloServerPluginLandingPageLocalDefault({ embed: false }),
+    // highlight-end
+  ],
 });
 ```
 
-For all available configuration options for these plugins, [see the API reference](../api/plugin/landing-pages/).
+For all available configuration options for these plugins, [see the API reference](/apollo-server/api/plugin/landing-pages/).
 
 ### GraphQL Playground
```

Comment:
Maybe we can drop this section (but keep around the section in the landing page plugins page)